runtime/commonMain/src/kotlinx/serialization/internal/LongAsStringSerializer.kt

```diff
@@ -4,17 +4,9 @@
 
 package kotlinx.serialization.internal
 
-import kotlinx.serialization.*
-
-object LongAsStringSerializer : KSerializer<Long> {
-    override val descriptor: SerialDescriptor =
-        PrimitiveDescriptor("kotlinx.serialization.LongAsStringSerializer", PrimitiveKind.STRING)
-
-    override fun serialize(encoder: Encoder, value: Long) {
-        encoder.encodeString(value.toString())
-    }
-
-    override fun deserialize(decoder: Decoder): Long {
-        return decoder.decodeString().toLong()
-    }
-}
+@Deprecated(
+    "Moved to kotlinx.serialization.builtins package",
+    ReplaceWith("LongAsStringSerializer", "kotlinx.serialization.builtins.LongAsStringSerializer"),
```

Comment:
This replacement does not work, unfortunately

Reply:
it works on `import kotlinx.serialization.internal.LongAsStringSerializer`, but not on usage. At least something